src/planscape/config/conditions_config.py

```diff
@@ -40,44 +40,32 @@ def check_regions(regionlist) -> bool:
 
         def check_region(region) -> bool:
             return (isinstance(region, dict) and
-                    region.keys() <= set(['region_name', 'pillars', 'filepath', 'display_name']) and
+                    region.keys() <= set(['region_name', 'pillars', 'filepath', 'display_name', 'colormap']) and
                     isinstance(RegionName(region['region_name']), RegionName) and
                     isinstance(region['pillars'], list) and
                     all([check_pillar(pillar) for pillar in region['pillars']]))
 
         def check_pillar(pillar) -> bool:
             return (isinstance(pillar, dict) and
-                    pillar.keys() <= {'pillar_name', 'elements', 'operation', 'filepath', 'display_name', 'display'} and
+                    pillar.keys() <= {'pillar_name', 'elements', 'operation', 'filepath', 'display_name', 'display', 'colormap'} and
                     isinstance(pillar['pillar_name'], str) and
                     isinstance(pillar['elements'], list) and
                     all([check_element(element) for element in pillar['elements']]))
 
         def check_element(element) -> bool:
             return (isinstance(element, dict) and
-                    element.keys() <= {'element_name', 'metrics', 'operation', 'filepath', 'display_name'} and
+                    element.keys() <= {'element_name', 'metrics', 'operation', 'filepath', 'display_name', 'colormap'} and
                     isinstance(element['element_name'], str) and
                     isinstance(element['metrics'], list) and
                     all([check_metric(metric) for metric in element['metrics']]))
 
         def check_metric(metric) -> bool:
             return (isinstance(metric, dict) and
-                    metric.keys() <= {'metric_name', 'filepath', 'display_name', 'current_conditions_only'} and
+                    metric.keys() <= {'metric_name', 'filepath', 'display_name', 'current_conditions_only', 'colormap'} and
                     isinstance(metric['metric_name'], str))
 
         return 'regions' in self._config and check_regions(self._config['regions'])
 
-    def get_regions(self) -> List[Region]:
```

Comment:
are we intentionally deleting this?

Reply:
Looks like there was a merge conflict.   See below.